dashboards/immersive-stream/metadata.yaml

```
@@ -0,0 +1,7 @@
+sample_dashboards:
+  -
+    category: Immersive Stream
+    id: StreamInstance
```

```suggestion
    id: immersive-stream
```

This needs to match the file names

---

I picked this ID to match the dashboard ID for the system dashboards. Updated to match the file names instead; I don't think there will be any issues with having the IDs be different.